src/main/kotlin/no/nav/tilleggsstonader/integrasjoner/brukervarsel/DittNavKafkaProducer.kt

```diff
@@ -0,0 +1,77 @@
+package no.nav.tilleggsstonader.integrasjoner.brukervarsel
+
+
+import no.nav.brukernotifikasjon.schemas.builders.BeskjedInputBuilder
+import no.nav.brukernotifikasjon.schemas.builders.NokkelInputBuilder
+import no.nav.brukernotifikasjon.schemas.builders.domain.PreferertKanal
+import no.nav.brukernotifikasjon.schemas.input.BeskjedInput
+import no.nav.brukernotifikasjon.schemas.input.NokkelInput
+import no.nav.tilleggsstonader.libs.log.SecureLogger.secureLogger
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.kafka.core.KafkaTemplate
+import org.springframework.stereotype.Service
+import java.net.URL
+import java.time.LocalDateTime
+import java.time.ZoneOffset.UTC
+
```

Comment:
Det ser ut som at ef-mottak ikke har blitt oppdatert helt med config for å unngå at man logger FNR ved feil.
Det burde registreres en bean for `LoggingProducerListener` som setter `includeContents` til false.

Dette burde også gjøres i en `@Configuration` som enabled Kafka. Eks
```kotlin
@EnableKafka
@Configuration
class KafkaConfig {

    @Bean
    fun loggingProducerListener() = LoggingProducerListener<Any, Any>().apply {
        setIncludeContents(false)
    }
```